packages/devextreme/scss/widgets/generic/gridBase/_colors.scss

```diff
@@ -464,7 +464,7 @@ $datagrid-text-stub-background-image-path: null !default;
 }
 
 @if $color == "light" {
-  $datagrid-columnchooser-item-color: lighten($datagrid-base-color, 38.5%) !default;
+  $datagrid-columnchooser-item-color: lighten($datagrid-base-color, 26%) !default;
```

Comment:
Let's improve other themes with percentages ("dark", "darkviolet", "greenmist") and cover this cases by matrix tests.

Reply:
Hello, further research suggests that:
- 'darkviolet' doesn't have the contrast issue
- Changing the color to #757575 or using the 26% solution doesn't work for 'greenmist'
- The color is not changed to #757575, but the 26% solution works in 'dark'